tests/testthat/test-FunctionReporter-class.R

```diff
@@ -6,12 +6,12 @@ rm(list = ls())
 ## PUBLIC INTERFACE ##
 
 test_that('FunctionReporter public interface is as expected', {
-
+    
```

Comment:
Can you please enable the following two formatting options for this project? They are found in Options > Code > Saving.

- Ensure that source files end with newline
- Strip trailing horizontal whitespace when saving

This ensures we won't have these sorts of extraneous whitespace changes.

Reply:
TIL.  I usually make my projects using `usethis::create_project` which has those set by default, but I guess creating the project within RStudio (File > New Project) does not do that.